transform/settings.py

```diff
@@ -19,7 +19,7 @@ def _get_value(key, default_value=None):
         raise ValueError()
     return value
 
-SDX_FTP_IMAGES_PATH = "EDC_QImages"
+SDX_FTP_IMAGE_PATH = "EDC_QImages"
```

Comment:
If we change this, does it need to be changed in gitlab?

Reply:
No , its set as a local variable not an environment variable . So as long as we change all accesses to ...IMAGE... not ...IMAGES... then we will be fine . The issue was that it WAS IMAGE on transform cs and IMAGES . By standardising then the index_file.py can be the same in both